### ISSUE TITLE
Unexpected token error since v0.49.15

### ISSUE THREAD

**Issue by r-plus** [nicklockwood/SwiftFormat#1253]
swift file

```swift
import UIKit

extension UIColor {
    var rgb: Int {
        var red: CGFloat = 0
        var green: CGFloat = 0
        var blue: CGFloat = 0
        var alpha: CGFloat = 0
        getRed(&red, green: &green, blue: &blue, alpha: &alpha)

        return lroundf(Float(red) * 255) << 16 | lroundf(Float(green) * 255) << 8 | lroundf(Float(blue) * 255)
    }
}
```

- no configuration file

result

```
$ mint run swiftformat test.swift --swiftversion 5.6
🌱 Finding latest version of swiftformat
🌱 Running swiftformat 0.49.16...
Running SwiftFormat...
error: Unexpected token ) at 11:76 in /tmp/sw/test.swift.
```

v0.49.14 will not emit no error for this file.

**Comment by raxityo** [comment]
I also ran into this recently in `v0.49.16`.
Workaround: Splitting the 3 colors into separate variables and performing the OR on those variables would make SwiftFormat happy.

**Comment by nicklockwood** [comment]
Thanks for the report - I actually found this yesterday and have already fixed it in the develop branch. I'll push a new release soon.

**Comment by nicklockwood** [comment]
Fixed in 0.49.17
